tenant_schemas/management/commands/__init__.py

```diff
@@ -86,27 +86,31 @@ class InteractiveTenantOption(object):
     def add_arguments(self, parser):
         parser.add_argument("-s", "--schema", dest="schema_name", help="specify tenant schema")
 
-    def get_tenant_from_options_or_interactive(self, **options):
+    def get_tenant_from_options_or_interactive(self, schema_name=None):
         TenantModel = get_tenant_model()
-        all_tenants = TenantModel.objects.all()
+        values = TenantModel.objects.values_list(
```

Comment:
Instead of `values_list()` why not use `only("schema_name", "domain_url")`? That way you can still keep the `(t.schema_name, t.domain_url,) for t in all_tenants])` because the way youre doing it now, the order can be reversed depending on the python version.